src/utils.c

```diff
@@ -128,6 +128,16 @@ char *get_album_cover_file(const char *location, const char *format) {
 	if (location == NULL)
 		return NULL;
 
+	/* show cover cue image file */
+	char *location_t;
+	location_t = strdup(location);
+
+	if (strncmp(location_t, "cue://", 6) == 0) {
+		strcpy(location_t, &location[6]);
+		location = dirname(location_t);
```

Comment:
Is this `dirname()` call needed here? Because there is another one on line 148. I just wonder whether it is required to strip two directories in order to get correct base path.

Reply:
```location: /home/andrey/Music/Alpha 430 - Purcell - King Arthur - Vox Luminis, Meunier - 2018/CD1/Purcell - King Arthur - CD1.cue/1```
after `dirname(location_t);` removes 1 at the end of the path (this is the track number in the cue file)

Reply:
OK I see, thanks.